build/pipelines/templates/build-console-fuzzing.yml

```diff
@@ -0,0 +1,21 @@
+parameters:
+  configuration: 'Fuzzing'
+  platform: ''
+  additionalBuildArguments: ''
+  minimumExpectedTestsExecutedCount: 1 # Sanity check for minimum expected tests to be reported
+  rerunPassesRequiredToAvoidFailure: 0

```

Comment:
```suggestion
```
We don't use these, so might as well remove them

Reply:
Yes, only needed for the Helix-post-processing step.